store-collected-images/README.md

```diff
@@ -0,0 +1,52 @@
+# "Image Reference Collector" example
+
+## What it does
+
+This example adds a context menu which target any image element in the webpage,
+when the context menu action is clicked, the extension open a window and add
+the related image element to the preview list of the collected images,
+the user can then store the collected images by giving the collection a name
+and pressing the **save** button.
```

Comment:
So, I'm horribly picky because of being a writer, but: this sentence is too long. Better:

This example adds a context menu which targets any image element in the webpage. When the context menu item is clicked, the add-on opens a window and adds the related image element to the preview list of the collected images. The user can then store the collected images by giving the collection a name
and pressing the **save** button.